src/use_case/kill_villager/KillVillagerDataAccessInterface.java

```diff
@@ -0,0 +1,12 @@
+package use_case.kill_villager;
+
+import entity.Game;
+import entity.PromptGenerator;
+
+public interface KillVillagerDataAccessInterface {
+    void save(Game game);
+    Game getGame();
+
+    PromptGenerator getPromptGenerator();
+    // getPromptGenerator in DAO should be pulling the ConversationHistory to then construct a PromptGenerator off of
```

Comment:
All good again just reminder that after we decide on the data access stuff in today's meeting you can probably delete this class